src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests5.cs

```diff
@@ -355,5 +367,234 @@ public static void Main()
                     //         _ = new C() is { Missing: null, Prop1.Prop2: {} };
                     Diagnostic(ErrorCode.ERR_FeatureInPreview, "Prop1.Prop2").WithArguments("extended property patterns").WithLocation(9, 41));
         }
+
+        [Fact]
+        public void ExtendedPropertyPatterns_SymbolInfo_01()
+        {
+            var source =
+@"
+using System;
+class Program
+{
+    public static void Main()
+    {
+        P p = new P();
+        Console.WriteLine(p is { X.Y: {}, Y.X: {} });
+    }
+}
+class P
```

Comment:
It would be better to split this type into two (one containing `X` and one for `Y`)